ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-openshift-stage.yaml

```
@@ -0,0 +1,75 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: amd64
+      product: ocp
+      version_bounds:
+        lower: 4.12.0-0
+        upper: 4.13.0-0
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: azure-upi-p3-f1-openshift-stage
+  cron: 0 0 * * 4
+  steps:
+    cluster_profile: azure-qe
+    env:
```

Instead of creating your own ref, just use the one that is already there for `install-operators`. It would look like this:
```suggestion
    env:
      BASE_DOMAIN: qe.azure.devcluster.openshift.com
      E2E_RUN_TAGS: '@azure-upi and @network-ovnkubernetes and not @fips and @stage and @stage_only'
      STAGE_INDEX_IMAGE: brew.registry.redhat.io/rh-osbs/iib-pub-pending:v4.12
      TAG_VERSION: '@4.12'
      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&;Stagerun&;~ProdrunOnly&;~VMonly&;~CPaasrunOnly&;~MicroShiftOnly&;~NonPreRelease&
      TEST_SCENARIOS: Stagerun
      OPERATORS: |
      [
          {"name": "sriov-network-operator", "source": "qe-app-registry", "channel": "4.13", "operator_group": "sriov-network-operators", "install_namespace": "openshift-sriov-network-operator", "target_namespaces": "openshift-sriov-network-operator"}
      ]
    test:
    - ref: enable-qe-catalogsource-stage
    - ref: install-operators
    - chain: openshift-e2e-test-stage-qe
    workflow: cucushift-installer-rehearse-azure-upi
```

Then you could delete the `step-registry/install-operators/install-sriov-network-operator` directory entirely